README.md

```
@@ -89,12 +96,12 @@
   <img src="./misc/wren-insight.png" style="max-width: 700px"/>
 </p>
 
-### 6. AI-Powered Insights & Visuals
+### 6. GenBI
 
-> AI-generated summaries that deliver key insights alongside SQL queries, making complex data easier to understand. Instantly transform results into AI-generated charts, turning raw data into clear, actionable visuals – empowering faster, smarter decisions.
+> The GenBI feature empowers users with AI-generated summaries that provide key insights alongside SQL queries, simplifying complex data. Instantly convert query results into AI-generated reports, charts, transforming raw data into clear, actionable visuals. With GenBI, you can make faster, smarter decisions with ease.
 
 <p align="center">
-  <img src="./misc/wren-visual.png" style="max-width: 700px"/>
+  <img src="./misc/wren-genbi.png" style="max-width: 700px"/>
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)